devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

```diff
@@ -1,184 +1,189 @@
-/**
- *  Copyright 2019 SmartThings
- *
- *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- *  in compliance with the License. You may obtain a copy of the License at:
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
- *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
- *  for the specific language governing permissions and limitations under the License.
- *
- */
-import physicalgraph.zigbee.zcl.DataType
-
-metadata {
-    definition (name: "Zigbee Metering Plug", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.smartplug", mnmn: "SmartThings",  vid: "generic-switch-power-energy") {

```

Comment:
Please fix the code so the diff here would show only Your changes.

Reply:
We tried various methods to fix it, unfortunately not working. This one is a re-submission after the first submission got the same problems

Reply:
@frient-design 
Maybe it's just a matter of line endings ?
https://docs.github.com/en/free-pro-team@latest/github/using-git/configuring-git-to-handle-line-endings